addOns/callhome/src/main/javahelp/org/zaproxy/addon/callhome/resources/help/contents/callhome.html

```diff
@@ -15,5 +15,8 @@ <H2>Check For Updates</H2>
 The Check for Updates call is used to find out the latest information about ZAP and the published add-ons. 
 It is a mandatory add-on from ZAP 2.12.0 as this feature is used by the ZAP core.
 
+<H2>News</H2>
+The News call is used to find out the latest new item about ZAP - this is displayed in the ZAP desktop. Quick Start panel 
```

Comment:
Could use a period.
Also displayed in launched browsers right?

Reply:
Uh, added the text at the wrong place. Pretty sure the new is not displayed in the launched browsers, but clicking on the news button does launch a browser..

Reply:
You're right, Browser Launch displays the manual explore message.

Reply:
Fixed the fullstop :)